JS_Control_Flow_homework/JS_Control_Flow_homework.js

```diff
@@ -0,0 +1,42 @@
+// let cap_cilindrica = prompt("introduceti capacitatea cilindrica");
+
+// while (isNaN(cap_cilindrica)) {
+//     cap_cilindrica = prompt("Introduceti o capacitate cilindrica valida:");
+// };
+// impozit = cap_cilindrica * 22 / 200;
+// impozit_final = Math.floor(impozit);
+// console.log("Impozitul datorat statului este: " + impozit_final);
+
+let auto_type = prompt("Care este tipul de auto pentru care vreti sa platiti impozit?(autoturism/motocicleta/camion");
```

Comment:
Aici mergea ca toate sa fie const nu let. Ele nu vad ca isi modifica valoarea in timp